crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -613,8 +616,8 @@ pub struct Data {
     pub capture: Option<String>,
     pub payment_url: Option<Url>,
     pub status: Option<MultisafepayPaymentStatus>,
-    pub error_code: Option<i32>,
-    pub error_info: Option<String>,
+    pub reason: Option<String>,
```

Comment:
Why do we need this api contract changes from PG?

Reply:
Utill this PR, multisafepay's 2xx response was not captured. `error_code` and `error_info` is sent in the 4xx response and these fields are not included in their 2xx response.